modules/test/playwright/tests/change-tracking-web/main/reviewChanges.spec.ts

```diff
@@ -324,3 +327,47 @@ test('LPD-52950 Assert publications user cannot see publications they do not hav
 
 	await performLoginViaApi({page, screenName: 'test'});
 });
+
+test('LPD-61747 Discarding changes in a Publication containing a deletion change throws NPE', async ({
+	changeTrackingPage,
+	ctCollection,
+	journalEditArticlePage,
+	journalPage,
+	page,
+}) => {
+	await journalEditArticlePage.goto();
+
+	const title = getRandomString();
+
+	await journalEditArticlePage.fillTitle(title);
+
+	await journalEditArticlePage.publishArticle();
+
+	await waitForAlert(page, `Success:${title} was created successfully.`);
+
+	await changeTrackingPage.workOnPublication(ctCollection);
+
+	await journalPage.goto();
+
+	await page.getByLabel(`Actions for ${title}`).click();
+
+	await page.getByRole('menuitem', {name: 'Delete'}).click();
```

Comment:
Is the issue reproducible with a deleted journal article? An NPE will only occur if the entity's `CTDisplayRenderer` uses the model in its `isHideable` logic. The test should fail before your fix is applied.